test/benchmark/tests/test_vgg16.py

```diff
@@ -23,6 +23,7 @@
 
 INSTANCE_TYPES_TO_TEST = ["p3.8xlarge"]
 
+@pytest.mark.skip()
```

Comment:
Is this being skipped?

Reply:
Yes, apologies, a leftover from testing. Unskipped.